src/model/constants/DraftBlockType.js

```diff
@@ -20,6 +20,9 @@ export type DraftBlockType = (
   'paragraph' |
   'header-one' |
   'header-two' |
+  'header-three' |
+  'header-four' |
+  'header-five' |
```

Comment:
Why stop at five? It just seems like an arbitrary choice between 1 and 6. Maybe a better choice than 2, but interested to know why you chose 5.